kolibri/plugins/device/assets/src/views/ManageContentPage/FilteredChannelListContainer.vue

```diff
@@ -1,29 +1,34 @@
 <template>
 
-  <div :class="{small: !windowIsLarge}">
-
-    <slot name="header"></slot>
-
-    <div class="top-panel">
-      <p class="count-msg" data-test="available">
-        {{ channelsCountMsg }}
-      </p>
-      <div v-if="channels.length > 0" class="filters">
-        <KSelect
-          v-model="languageFilter"
-          class="filter-lang"
-          :options="languageFilterOptions"
-          :label="$tr('languageFilterLabel')"
-          :inline="true"
-        />
-        <FilterTextbox
-          v-model="titleFilter"
-          class="filter-title"
-          :placeholder="$tr('titleFilterPlaceholder')"
-          :throttleInput="500"
-        />
-      </div>
-    </div>
+  <div :class="{'fclc-sm': !windowIsLarge}">
+    <KGrid class="top-panel">
+      <KGridItem :layout12="{span: 4}">
```

Comment:
should be span 12, not 4:

![image](https://user-images.githubusercontent.com/2367265/69153992-313e2e00-0aad-11ea-8822-d5f4b64faaa9.png)